src/main/c/waf_jni.c

```diff
@@ -78,7 +79,6 @@ static void _throw_pwaf_exception(JNIEnv *env, DDWAF_RET_CODE retcode);
 static void _throw_pwaf_timeout_exception(JNIEnv *env);
 static void _update_metrics(JNIEnv *env, jobject metrics_obj, const ddwaf_result *ret);
 static bool _convert_ddwaf_config_checked(JNIEnv *env, jobject jconfig, ddwaf_config *out_config);
-static void _dispose_of_ddwaf_config(ddwaf_config *cfg);
```

Comment:
This is still needed to free up resources after passing ddwaf_config to the builder. Similar to what was done before.